mvp.feature

```diff
@@ -830,20 +830,21 @@ Feature: MVP features for NVA
       | Suggestions from ORCID |
 
   @354
-  Scenario Outline: Creator opens My Publications
-    Given that the user is logged in as Creator
+  Scenario: Creator opens My Publications
+    Given the user is logged in as Creator
     When they click the menu item My Publications
-    Then they see My Publications
-    And they see a list of all unpublished registrations with the fields
+    Then the page My Publications is opened
+    And a list of all unpublished registrations with the fields
       | Title    |
       | <Status> |
       | Created  |
-    And they see each list item has a button Delete and Edit that is enabled
-
     Examples:
       | Status   |
       | Draft    |
       | Rejected |
+    And each list item has a button Delete and Edit that is enabled
```

Comment:
The statement should be above the examples, the scenario outline runs in relation to the examples.